stream.js

```diff
@@ -73,8 +73,14 @@ function WebSocketStream(target, protocols, options) {
   if (socket.readyState === socket.OPEN) {
     stream = proxy
   } else {
-    stream = duplexify.obj()
-    socket.onopen = onopen
+    if (isBrowser) {
+      stream = proxy
+      stream.cork()
+      socket.onopen = onopenBrowser
```

Comment:
can the same logic be applied to Node as well? What will cause issues?

Reply:
I'll try to use same logic for Node and browser, but for Node needed duplexify logic for destroy() method as I see.